products/sle/main.pm

```diff
@@ -726,19 +726,33 @@ elsif (get_var("QA_TESTSUITE")) {
     loadtest "qa_automation/install_test_suite";
     loadtest "qa_automation/execute_test_run";
 }
-elsif (get_var("XFSTESTS")) {
-    prepare_target;
-    if (check_var('XFSTESTS', 'installation')) {
-        loadtest "xfstests/install";
+elsif (get_var('XFSTESTS')) {
+    if (get_var('FSTESTS_SPVM')) {
```

Comment:
why not checking for BACKEND==spvm ? No need for a new extra variable.

Reply:
Because I don't know this parameter, I'll change to use this exist one, thanks :)

Reply:
there is even function `is_spvm`